glTFRevitExport/Extensions/RevitAPIExtensions.cs

```diff
@@ -137,5 +137,24 @@ public static bool IsBIC(this Category c, BuiltInCategory bic)
                 z: vector[2]
             );
         }
+
+        public static bool CurveIntersects(this BoundingBoxXYZ bbox, Curve curve)
+        {
+            if (bbox == null || curve == null)
+            {
+                return false;
+            }
+
+            var startPoint = curve.Evaluate(0, true);
+            var endPoint = curve.Evaluate(1, true);
+
+            var b1 = new Vector3D((float)bbox.Min.X, (float)bbox.Min.Y, (float)bbox.Min.Z);

```

Comment:
maybe we could create a method that returns a Vector3D from the bounding box. 
Vector3D CreateVectorFromBBox(BoundingBox bbox)

Reply:
perfect! Done!